docs/examples/advanced_http.py

```diff
@@ -0,0 +1,12 @@
+from nameko.web import http
+
+
+class Service(object):
+
+    @http('GET', '/404')
+    def does_not_exist(self, value):
+        return 404, "Does Not Exist"
```

Comment:
I'm not sure I'd use this as an example, as `does_not_exist` clearly does exist and deliberately returns a 404 – it's not a fallback hander. Perhaps a 204 or 403 instead?